include/MidiPort.h

```diff
@@ -174,6 +174,8 @@ private slots:
 	Map m_readablePorts;
 	Map m_writablePorts;
 
+	QMap<int16_t, int16_t> m_activeMIDIPitch;
```

Comment:
In the interest of not using Qt within the core, ``m_activeMidiPitch`` should be a ``std::unordered_map``